crp_APL/controllers/ctrl_vehicle_control_lat_stanley/include/ctrl_vehicle_control_lat_stanley/ctrl_vehicle_control_lat_stanley.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef CRP_APL_CTRL_VEHICLE_LAT_STANLEY_HPP
+#define CRP_APL_CTRL_VEHICLE_LAT_STANLEY_HPP
+
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <math.h>
+#include "rclcpp/rclcpp.hpp"
+#include "geometry_msgs/msg/pose.hpp"
+#include "std_msgs/msg/float32_multi_array.hpp"
+#include "std_msgs/msg/float32.hpp"
+#include "crp_msgs/msg/ego.hpp"
+
+#include "autoware_control_msgs/msg/control.hpp"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "autoware_vehicle_msgs/msg/steering_report.hpp"
+
+#include "ctrl_vehicle_control_lat_stanley/controller_inputs.hpp"
+#include "ctrl_vehicle_control_lat_stanley/controller_outputs.hpp"
+
+namespace crp
+{
+    namespace apl
+    {
+        class CtrlVehicleControlLat : public rclcpp::Node
```

Comment:
Same package-class name inconsistency here: CtrlVehicleControlLatStanley

Reply:
it hink its fine like that -> Lateral Controller, and it fits the purpose
have a felling that **CtrlVehicleControlLatStanley** naming would be a bit messy..